docs/source/customization.mdx

```diff
@@ -46,171 +46,178 @@ else:
 Consult the 🤗 Accelerate [documentation](https://huggingface.co/docs/accelerate/usage_guides/deepspeed) for more information about the DeepSpeed plugin.
 
 
-## Use different optimizers
+## Use different optimizers and schedulers
```

Comment:
Lots of this doc come from the time when we just had `PPOTrainer` which wasn't a subclass of `transformers`. I think we should remove anything that can be achieved on the `transformers` side, perhaps with an opening comment like:

"Every trainer in TRL is a subclass of the `Trainer` from 🤗 Transformers. This means you can change the optimizer or learning rate scheduler by simply changing the corresponding attributes of the TRL trainer configuration class. See the 🤗 Transformers [documentation](https://huggingface.co/docs/transformers/perf_train_gpu_one#optimizer-choice) for more details."

Note we can also nuke the deepspeed doc above since that's a relic from the old PPO days

Reply:
Agree with you. Let's apply these recommendation in other PRs (see https://github.com/huggingface/trl/pull/2174#issuecomment-2399843454)